src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/infrastructure/client/util/HomeOfficeDateFormatter.java

```diff
@@ -17,4 +17,12 @@ public static String getCurrentDateTime() {
             DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss'Z'"));
     }
 
+    public static String getIacDecisionDate(String hoDecisionDate) {
+        return (hoDecisionDate != null && hoDecisionDate.length() == 20)
+            ? hoDecisionDate.substring(8, 10)
+            + "/" + hoDecisionDate.substring(5, 7)
+            + "/" + hoDecisionDate.substring(0, 4)
+            : "";
```

Comment:
Is it an expected scenario to have the date empty?

Reply:
If we do not receive a proper date from Home office, we do not want to throw exception, so setting it to ""

Reply:
I am formatting the date. If there is any exception, then we return the data as sent by Home office. Confirmed with Damian and Ana